### DISCUSSION TITLE
Ruskdesk's CPU is higher than Anydesk's

### DISCUSSION THREAD

**Discussion by bauhuynhdev** [rustdesk/rustdesk#5703] [General]
Currently, Rustdesk's CPU is higher than Anydesk's, is there any way to fix it?

Rustdesk's CPU
Rustdesk comes with Desktop Window Manager and Anydesk does not.
![image](https://github.com/rustdesk/rustdesk/assets/22582143/3fac1de4-13ef-433b-96da-ec5bb5a4f7e8)

Anydesk's CPU
![image](https://github.com/rustdesk/rustdesk/assets/22582143/2de289ef-0228-45e4-b1e1-111900fc1059)

**Comment by 21pages** [comment]
try https://github.com/rustdesk/rustdesk/discussions/5264
